terminal_api_flows/flows/credit_txn.py

```diff
@@ -5,10 +5,85 @@
 from terminal_api_flows.tools.decorators import terminal_ping_decorator_3_attempts
 
 
+CARD_DECLINE_STATUSES = [
+    "NFC_READ_ERROR",
+    "DIP_READ_ERROR",
+    "SWIPE_READ_ERROR",
+]
```

Comment:
This is not catching everything. I believe we need all these like the ticket shows:
```
DECLINED_ERROR, DECLINED_ERROR_DIP_MSG, DECLINED_ERROR_SWIPE_MSG, PAYMENT_GENERAL_ERROR, PAYMENT_GENERAL_DIP_MSG, PAYMENT_GENERAL_SWIPE_MSG, DIP_READ_ERROR, DIP_READ_ERROR_MSG, SWIPE_READ_ERROR, SWIPE_READ_ERROR_MSG, NFC_READ_ERROR, NFC_DECLINE_ERROR_MSG, NFC_GENERAL_ERROR_MSG
```